config/config_mgmt.py

```diff
@@ -71,18 +68,24 @@ def __init__(self, source="configDB", debug=False, allowExtraTables=True):
 
         return
 
-    def logInFile(self, header="", obj=None, json=False):
+    def logInFile(self, msg=None, logLevel='INFO'):
+
+        logLevelMap= {'CRIT'  : syslog.LOG_CRIT,
+                      'ERR'   : syslog.LOG_ERR,
+                      'INFO'  : syslog.LOG_INFO,
+                      'DEBUG' : syslog.LOG_DEBUG
+                   }
+        # return if loglevel is DEBUG. but debug is not set
+        if logLevel == 'DEBUG' and self.DEBUG == False:
```

Comment:
why do we need self.DEBUG now? I think we should be able to enable the log level and use the priority mask